src/main/java/seedu/contax/ui/MainWindow.java

```diff
@@ -152,31 +190,29 @@ private void setWindowDefaultSize(GuiSettings guiSettings) {
     }
 
     /**
-     * Changes the type of model being displayed in the content list.
-     *
-     * @param contentType The type of content the UI should display.
+     * Changes the selected tab from the specified {@code contentType}.
      */
-    private void changeListContentType(GuiListContentType contentType) {
+    private void changeTabFocus(GuiListContentType contentType) {
         if (contentType == null) {
-            contentListPanelPlaceholder.getChildren().clear();
+            tabbedPanelPlaceholder.getTabs().clear();
```

Comment:
You should clear focus, not clear tabs.

Reply:
Or you can ignore, since you cannot exactly clear the focus away.

Reply:
Fixed in ae77346. Removed null checker.